joblib/test/test_memory.py

```diff
@@ -229,8 +229,8 @@ def name_collision(x):
         a(1)
         b(1)
 
-        yield assert_equal, len(w), 1
-        yield assert_true, "collision" in str(w[-1].message)
+        assert len(w) == 1
```

Comment:
Don't worry this was a side comment more than anything else. I am not asking for the code to be changed.